crates/ruff_dev/Cargo.toml

```diff
@@ -11,6 +11,7 @@ clap = { workspace = true }
 itertools = { workspace = true }
 libcst = { workspace = true }
 once_cell = { workspace = true }
+pretty_assertions = { version = "1.3.0" }
```

Comment:
i'm surprised i didn't find a better `git diff` like library but that's good enough for this case